pkg/solana/config/config.go

```diff
@@ -0,0 +1,152 @@
+package config
+
+import (
+	"sync"
+	"time"
+
+	"github.com/gagliardetto/solana-go/rpc"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/db"
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/logger"
+)
+
+// Global terra defaults.
+var defaultConfigSet = configSet{
+	BlockRate:           500 * time.Millisecond, // approximate slot rate
```

Comment:
should this be slowed down? this is tied to how often the balance monitor queries for updated balance

Reply:
ah ok, so better to rename this to BalancePollPeriod. we don't really use the BlockRate parameter at all besides polling for balances (ported from Terra)

Reply:
renamed parameter, and changed to 5s poll rate: https://github.com/smartcontractkit/chainlink-solana/pull/250/commits/b7c508bd4823b4d7a58c376b1b98d3983ce45101